.travis/build_examples.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+repo_path="$GOPATH/src/github.com/unidoc/unipdf-examples"
+git clone https://github.com/unidoc/unipdf-examples.git $repo_path
+cd $repo_path
+
+branch_name="v3"
+if [[ `git ls-remote origin "$TRAVIS_BRANCH"` ]]; then
+    branch_name="$TRAVIS_BRANCH"
+fi
+
+git checkout $branch_name
+./build_examples.sh
```

Comment:
I noticed in the build output there was:
```
$ ./.travis/build_examples.sh
Cloning into '/home/travis/gopath/src/github.com/unidoc/unipdf-examples'...
remote: Enumerating objects: 52, done.
remote: Counting objects: 100% (52/52), done.
remote: Compressing objects: 100% (44/44), done.
Receiving objects:  63% (remote: Total 928 (delta 16), reused 27 (delta 8), pack-reused 876
Receiving objects: 100% (928/928), 3.88 MiB | 14.76 MiB/s, done.
Resolving deltas: 100% (601/601), done.
Already on 'v3'
Your branch is up to date with 'origin/v3'.
Building to bin/ folder
go: finding github.com/youtube/vitess v2.1.1+incompatible
go: finding github.com/wcharczuk/go-chart v2.0.1+incompatible
go: finding github.com/miekg/pkcs11 v1.0.2
go: finding github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0
go: finding github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
go: finding github.com/blend/go-sdk v2.0.0+incompatible
go: finding github.com/ThalesIgnite/crypto11 v0.1.0
go: finding github.com/unidoc/unipdf/v3 v3.1.1
go: downloading github.com/unidoc/unipdf/v3 v3.1.1
...
```
which indicates it was downloading the unipdf from the go.mod file.

It might be necessary to add a replace directive to the go.mod file, something like
```
replace github.com/unidoc/unipdf/v3 => /path/to/github.com/unidoc/unipdf
```
This can also be added with the go command line tool (see `go help mod edit`).